### DISCUSSION TITLE
How to...

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#11866] [Q&A]
Is there any way to see the covers of books in black & white not in color by turning some button on or off? I have a color E-Reader.
Not all the time, but sometimes by button, when I want (cause some covers are bad in 100 dpi color resolution)

**Comment by jonnyl2** [comment]
A bit of a workaround, but you could switch off color rendering (Top menu -> Gear -> Screen -> Color rendering), go to the book page showing the cover, take a screenshot and select `Set as book cover`, and then reenable color rendering. 

Tested on an Android device. 

Edit: But it probably won't help you on a color e-ink screen because the screen will still display them as color images with low resolution AFAIK.

**Reply by ghost** [reply]
> A bit of a workaround, but you could switch off color rendering (Top menu -> Gear -> Screen -> Color rendering), go to the book page showing the cover, take a screenshot and select `Set as book cover`, and then reenable color rendering. 
> 
> Tested on an Android device. 
> 
> Edit: But it probably won't help you on a color e-ink screen because the screen will still display them as color images with low resolution AFAIK. 

I tried this button, but nothing changes. 

I wish there was a b&w option (function) in KOReader
